src/uct/self/self_md.c

```diff
@@ -45,7 +54,7 @@ static ucs_status_t uct_self_md_open(const char *md_name, const uct_md_config_t
 
 UCT_MD_COMPONENT_DEFINE(uct_self_md, UCT_SELF_NAME,
                         uct_self_query_md_resources, uct_self_md_open, NULL,
-                        ucs_empty_function_return_unsupported,
-                        ucs_empty_function_return_unsupported, "SELF_",
+                        ucs_empty_function_return_success,
```

Comment:
@yosefe I see this handle_p became uct_rkey_bundle_t::handle by uct_rkey_unpack. This handle used internally in uct_<TL>_rkey_release (loopback doesn't use these funcs).

Should I implement some stub function to set this handle_p to NULL (as IB does)?

if this a bug, I found the same here https://github.com/openucx/ucx/blob/master/src/uct/sm/cma/cma_md.c#L75
Should I do the same for SM:CMA?